src/redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+const STATUS = 'STATUS';
+const INITIAL_STATE = [];
+
+export function checkStatus(id) {
+  return {
+    type: STATUS,
+    payload: id,
```

Comment:
- [ ] **[OPTIONAL]** I don't think you would be needing the `id` and `payload` here. I would recommend you remove them.

Reply:
ok thanks I will